RecDP/pyrecdp/primitives/llmutils/rag_data_loader.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+
+from pyrecdp.core.utils import Timer
+from pyrecdp.primitives.operations import DirectoryLoader, DocumentSplit, DocumentIngestion
+
+
+def load_data_to_vs(data_dir, out_dir, index="doc_store"):
+    from pyrecdp.LLM import TextPipeline
+    text_splitter = "RecursiveCharacterTextSplitter"
+    text_splitter_args = {"chunk_size": 500, "chunk_overlap": 0}
+    pipeline = TextPipeline()
+    ops = [
+        DirectoryLoader(data_dir, glob="**/*.pdf"),
```

Comment:
Shall we use '‘Html2md_Loader’? For better support since PDF is not fully working yet.
I'll schedule a meeting with you to come up with a better idea.